### DISCUSSION TITLE
@types/cheerio has own types as of v1.0.0-rc.9

### DISCUSSION THREAD

**Discussion by moltar** [DefinitelyTyped/DefinitelyTyped#69639] [Issues with a @types package]
https://github.com/cheeriojs/cheerio/releases/tag/v1.0.0-rc.9

Need to update in https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/notNeededPackages.json

[a comment by an automated account is not shown]

**Comment by hkleungai** [comment]
I have opened PR https://github.com/DefinitelyTyped/DefinitelyTyped/pull/72495 for your request.
